### PR TITLE
Ms user profile

### PR DESCRIPTION
**Pull request by MayaS1111**
Created user profile that shows user's own posts and comments for each post

### CODE REVIEW

File: app/views/comments/_form.html.erb [hunk 1]
```diff
@@ -11,22 +11,12 @@
...
 
-  <div>
-    <%= form.label :author_id, style: "display: block" %>
-    <%= form.text_field :author_id %>
-  </div>
-
-  <div>
-    <%= form.label :body, style: "display: block" %>
-    <%= form.text_area :body %>
+  <%= form.hidden_field :photo_id %>
```
Comment (JoseMDO) [inline app/views/comments/_form.html.erb:14]: nice use of hidden inputs!

File: config/routes.rb [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
   root "photos#index"
+
   devise_for :users
   resources :likes
   resources :follow_requests
   resources :comments
   resources :photos   
+  #resources :users, only: :show
   
+  get "/:username" => "users#show", as: :user
```
Comment (JoseMDO) [inline config/routes.rb:11]: nice!

**Review by JoseMDO — APPROVED** [review]
Great job on your PR Maya!

**Review by jb520 — APPROVED** [review]
Everything looks good to me!

**Pull request merged**